app/services/moves/finder.rb

```diff
@@ -9,7 +9,7 @@ def initialize(filter_params)
     end
 
     def call
-      apply_filters(Move).order('locations.title')
+      apply_filters(Move).order('moves.id')
```

Comment:
Is this the default sort? Wondering if this would do the same?

```ruby
def call
  apply_filters(Move)
end
```

Reply:
I don't get an `ORDER BY` clause in a quick test. (`apply_filters(Move).to_sql` gave me something like `SELECT moves.* FROM moves WHERE moves.from_location_id = 'a11f6867-bd35-409e-a034-7f64a8b91c13'`)